src/services/homework/model.js

```diff
@@ -11,22 +11,24 @@ const { Schema } = mongoose;
 const homeworkSchema = new Schema({
 	schoolId: { type: Schema.Types.ObjectId, required: true },
```

Comment:
i would not set archived and private but schoolId instead which should reduce the list by a lot.

Reply:
ok.